### PR TITLE
Add params to flow starts API

### PR DESCRIPTION
**Pull request by norkans7**
Addresses https://github.com/rapidpro/rapidpro/issues/1148

### CODE REVIEW

File: temba/api/v2/serializers.py [hunk 1]
```diff
@@ -1009,6 +1015,10 @@ def save(self):
         restart_participants = self.validated_data.get("restart_participants", True)
         extra = self.validated_data.get("extra")
 
+        params = self.validated_data.get("params")
+        if params:
```
Comment (norkans7) [inline temba/api/v2/serializers.py]: If both `extra` and `params` are given we use `params`

File: temba/api/v2/views.py [hunk 2]
```diff
@@ -3150,7 +3150,7 @@ class FlowStartsEndpoint(ListAPIMixin, WriteAPIMixin, BaseAPIView):
      * **groups** - the list of groups that were started in the flow (objects)
      * **restart_particpants** - whether the contacts were restarted in this flow (boolean)
      * **status** - the status of this flow start
-     * **extra** - the dictionary of extra parameters passed to the flow start (object)
+     * **params** - the dictionary of extra parameters passed to the flow start (object)
```
Comment (norkans7) [inline temba/api/v2/views.py]: Docs only mention `params` however extra is still returned for reads and supported for writes

**Comment by nicpottier** [comment]
Let's add `params` on the read as well just for consistency.

**Comment by norkans7** [comment]
@nicpottier `params` was already added on the read

**Pull request merged**